component/org.wso2.carbon.identity.api.otp.service.emailotp/src/gen/java/org/wso2/carbon/identity/api/otp/service/emailotp/dto/OTPGenerateResponse.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com).
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.api.otp.service.emailotp.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import javax.validation.constraints.*;
+
+
+import io.swagger.annotations.*;
+import java.util.Objects;
+import javax.validation.Valid;
+import javax.xml.bind.annotation.*;
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class OTPGenerateResponse  {
+  
+    private String transactionId;
+    private String emailOtp;
+
+    /**
+    * Unique identifier of the generated OTP
+    **/
+    public OTPGenerateResponse transactionId(String transactionId) {
+
+        this.transactionId = transactionId;
+        return this;
+    }
+    
+    @ApiModelProperty(value = "Unique identifier of the generated OTP")
+    @JsonProperty("transactionId")
+    @Valid
+    public String getTransactionId() {
+        return transactionId;
+    }
+    public void setTransactionId(String transactionId) {
+        this.transactionId = transactionId;
+    }
+
+    /**
+    * The generated OTP
+    **/
+    public OTPGenerateResponse emailOtp(String emailOtp) {
+
+        this.emailOtp = emailOtp;
+        return this;
+    }
+    
+    @ApiModelProperty(value = "The generated OTP")
+    @JsonProperty("emailOtp")
+    @Valid
+    public String getEmailOtp() {
+        return emailOtp;
+    }
+    public void setEmailOtp(String emailOtp) {
+        this.emailOtp = emailOtp;
+    }
+
+
+
+    @Override
+    public boolean equals(java.lang.Object o) {
```

Comment:
Better if we can have meaningful parameter names

Reply:
`auto-generated` files.